src/index.js

```diff
@@ -559,28 +559,29 @@ class DataProvider {
   };
 
   storeResourcesData = (resourcesData, resourceNames, resources) => {
-    resourcesData.forEach(({ data: resourceData }, index) => {
+    resourcesData.forEach(({ data: resourceData, total: resourceTotal }, index) => {
       const resourceName = resourceNames[index];
       const resource = resources[resourceName];
       if (!Array.isArray(resourceData)) {
         resource.data = [resourceData];
       } else {
         resource.data = resourceData;
       }
+      resource.total = resourceTotal;
```

Comment:
Why do we want to store resource total here?

Reply:
`react-admin` expects the data provider to return an object with `{data, total}` from `getList` for the pagination to work. 
https://marmelab.com/react-admin/doc/3.1/DataProviders.html#response-format
Since we are returning the result of `aggregateData(resources)` in `handleGetQueries`, we have to store the total on the `resources` object if we want to be able to return it.
https://github.com/dryhten/ra-resource-aggregator/blob/f94e2e524177f3de82538a768f235236232ddaf1/src/index.js#L399

Reply:
Yes, I am aware react-admin expects `{data, total}` for `getList`. What is wrong with using the length property on the array, like it was before? `handleGetQueries` returns the aggregated data, which is an array of all the records, so the total number of records is the length of the array if that makes sense?

Reply:
see below https://github.com/dryhten/ra-resource-aggregator/pull/20#discussion_r402388100